src/material/schematics/ng-update/migrations/theming-api-v12/theming-api-migration.ts

```diff
@@ -22,7 +22,8 @@ export class ThemingApiMigration extends DevkitMigration<null> {
     if (extname(stylesheet.filePath) === '.scss') {
       const content = stylesheet.content;
       const migratedContent = content ? migrateFileContent(content,
-        '~@angular/material/', '~@angular/cdk/', '~@angular/material', '~@angular/cdk') : content;
+        '~@angular/material/', '~@angular/cdk/', '~@angular/material', '~@angular/cdk',
+        undefined, /material\/prebuilt-themes|cdk\/.*-prebuilt/) : content;
```

Comment:
I think we should also capture the obviously uncommon case of `material/core/theming/prebuilt`. IIRC we recommended that in the past (when Gulp was still used for releasing). e.g. https://stackoverflow.com/questions/43436779/how-to-include-theming-the-angular-2-material-in-webpack-app-jhipster/43751541

Reply:
Those files don't exist in the npm package though. The only prebuilt `.css` files I found were the ones in the regex above.

Reply:
Ops. Good point. I was thinking they exist there. Nevermind this comment then